source/fundamentals/versioned-api.txt

```diff
@@ -0,0 +1,123 @@
+=============
+Versioned API
+=============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. note::
+
+   The Versioned API feature requires MongoDB Server 5.0 or later. You
+   should only use the Versioned API feature if you are connecting to
+   MongoDB servers that are upgraded to a version that supports this feature.
+
+This guide shows you how to specify the **Versioned API** when connecting to
+a MongoDB instance or replica set. You can use the Versioned API feature to
+force the server to run operations with consistent behavior as specified by
+an **API version**. An API version defines expected behavior of operations
+such as the information expected in the request and the response.
```

Comment:
I would say "parameters" and format. Parameters is in quotes because it's not just the method signature (user parameters), but also how that data gets shared by the driver/server (Presentation and Application Layers).
I'm not sure how to improve upon how I've generalized this, but am open to suggestions.